### DISCUSSION TITLE
Does example build work on xcode 16.2 16C5032a ?

### DISCUSSION THREAD

**Discussion by skurgansky-sugarcrm** [jpudysz/react-native-unistyles#424] [Q&A]
I have being using unistyles beta.1 on my project. But today after update on xcode + simulator i got an error in build time. I see that example app doesn't work as well ?

**Comment by jpudysz** [comment]
Yesterday I saw first issues with Xcode 16.2. Ppl report that it doesn't work, so I guess Xcode 16.2 requires some extra work to bump `ios` compilation version to 16.0.

**Reply by skurgansky-sugarcrm** [reply]
i use expo with [plugin](https://docs.expo.dev/versions/latest/sdk/build-properties/#example-appjson-with-config-plugin) 
```json
[
       "expo-build-properties",
       {
         "ios": {
           "deploymentTarget": "16.6" // 17.0 or 18.0 none of this worked for me
         }
       }
     ]
```
minimum deployment target is updated successfully 
![image](https://github.com/user-attachments/assets/5d1512ff-6763-48c4-b652-d5212ec30623)

**Comment by skurgansky-sugarcrm** [comment]
https://github.com/swiftlang/swift/pull/77979
